### PR TITLE
modified local routes for cymru and naidheachdan article pages

### LINKED ISSUE #4343: Update cypress routes for
**Is your feature request related to a problem? Please describe.**
Update cypress services config to use the new routes for /cymrufyw & /naidheachdan setup as a part of this PR https://github.com/bbc/simorgh/pull/4270

**Describe the solution you'd like**
Update cypress config for local test for cymrufyw & 

`/cymrufyw/articles` > `/cymrufyw/erthyglau`

`/naidheachdan/articles` > `/naidheachdan/sgeulachdan`

This should be done for local only, until the mozart and STM routes are updated

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### LINKED ISSUE #4557: Navigation links not working properly for variants
**Is your feature request related to a problem? Please describe.**
Right now navigation links on services with variants (ukchina, zhongwen, serbian) are only working on frontpages. E.g. if you are on a `ukchina/trad` homepage and you click on any navigation link, you get redirected to a url on the correct variant. However, if you are not on the homepage but on a topic page (e.g. on `ukchina/trad/sport`) and click on `ukchina/trad/video` then you get redirected to `ukchina/simp/video`.

**Describe the solution you'd like**

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by PriyaKR**
Resolves #4343 

**Overall change:** _Updated cypress `services.js` config to use the new routes for /cymrufyw & /naidheachdan._

**Code changes:**

- Path changes in `services.js` from _`/cymrufyw/articles` > `/cymrufyw/erthyglau`_

- _`/naidheachdan/articles` > `/naidheachdan/sgeulachdan`_

Done for local only, as the mozart and STM routes aren't updated_
---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by oluoluoxenfree — APPROVED** [review]

File: cypress/support/config/services.js [hunk 1]
```diff
@@ -705,14 +707,16 @@ const services = {
         path:
           Cypress.env('APP_ENV') === 'live'
             ? undefined
-            : '/naidheachdan/articles/c18q7nedn2ko',
+            : Cypress.env('APP_ENV') === 'test'
+            ? '/naidheachdan/articles/c18q7nedn2ko'
+            : '/naidheachdan/sgeulachdan/c18q7nedn2ko',
```
Comment (sareh) [inline cypress/support/config/services.js:708]: We might wish to not test against the `test` env until the urls are corrected. So I'd suggest changing this logic to this:
```
          Cypress.env('APP_ENV') === 'live' || Cypress.env('APP_ENV') === 'test'
            ? undefined
            : '/naidheachdan/sgeulachdan/c18q7nedn2ko',
```

File: cypress/support/config/services.js [hunk 2]
```diff
@@ -277,14 +277,16 @@ const services = {
         path:
           Cypress.env('APP_ENV') === 'live'
             ? undefined
-            : '/cymrufyw/articles/c06p32z9x2mo',
+            : Cypress.env('APP_ENV') === 'test'
+            ? '/cymrufyw/articles/c06p32z9x2mo'
+            : '/cymrufyw/erthyglau/c06p32z9x2mo',
```
Comment (sareh) [inline cypress/support/config/services.js:280]: Same for this case
```
          Cypress.env('APP_ENV') === 'live' || Cypress.env('APP_ENV') === 'test'
            ? undefined
            : '/cymrufyw/erthyglau/c06p32z9x2mo',
```

File: cypress/support/config/services.js (hunk 1 above)
Comment (PriyaKR) [reply cypress/support/config/services.js:708]: Sounds good.

**Review by chris-hinds — APPROVED** [review]

**Review by sareh — APPROVED** [review]
👍 Looks good

**Pull request merged**